queries/synths/useFeeReclaimPeriodsQuery.ts

```diff
@@ -52,7 +52,7 @@ const useFeeReclaimPeriodsQuery = (
 			const waitingPeriodsRaw = (await ethcallProvider.all(
 				waitingPeriodCalls
 			)) as ethers.BigNumberish[];
-			const feesRaw = (await ethcallProvider.all(feeCalls)) as [any, any, any];
+			const feesRaw = (await ethcallProvider.all(feeCalls)) as [any, any, any]; // TODO: replace any with correct iterable type
```

Comment:
Sorry about the confusion here as well. Tested it locally and it should be:
```suggestion
			const feesRaw = (await ethcallProvider.all(feeCalls)) as [ethers.BigNumber, ethers.BigNumber, ethers.BigNumber][];
```

Reply:
will fix